eng/scripts/Get-PackageVersions.ps1

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env pwsh
+
+#Requires -Version 7.0
+[CmdletBinding()]
+param(
+  [switch]$Dependencies
```

Comment:
Can we rename this to something else that is "booleanish", like "CheckDependencies" or something? I was surprised to see it was a `[switch]` and had to look for what it does. So it wasn't intuitive to know whether I should use it or not.

Reply:
It's a switch that changes the shape of the output.  Like `-AsHashtable`  or `-Raw`
I think I prefer it as a separate script.